SimTracker/SiPixelDigitizer/plugins/PreMixingSiPixelWorker.cc

```diff
@@ -149,9 +160,35 @@ void PreMixingSiPixelWorker::addPileups(PileUpEventPrincipal const& pep, edm::Ev
   edm::Handle<edm::DetSetVector<PixelDigi>> inputHandle;
   pep.getByLabel(pixeldigi_collectionPile_, inputHandle);
 
+  // added for the Late CR
+  edm::Handle<edm::DetSetVector<PixelSimHitAddExtraInfo>> pixelAddInfo;
+  pep.getByLabel(pixeldigi_extraInfo_, pixelAddInfo);
```

Comment:
`getByLabel` is deprecated: https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideEDMGetDataFromEvent#get_ByLabel_deprecated  
This a priori cannot even work at runtime? How was it tested?

Reply:
In fact, if I change getByLabel by getByToken, the compilation crashes because 'const class PileUpEventPrincipal' has no member named 'getByToken'. see [https://github.com/cms-sw/cmssw/blob/master/SimGeneral/MixingModule/interface/PileUpEventPrincipal.h#L33](the use of getByLabel in PileUpEventPrincipal.h).

Reply:
whoops, sorry I thought it was an event data call (which it isn't). Disregard the comment then.